app/components/content/content.js

```diff
@@ -255,6 +255,14 @@ const Content = ({ p2p, contentKey: key, version, renderRow }) => {
             onClick={() => {
               remote.shell.openPath(`${directory}/${content.rawJSON.main}`)
             }}
+            draggable='true'
```

Comment:
```suggestion
            draggable
```

in HTML, just the attribute being there makes it `true` (can also be changed below)